Intermittent NU3004 errors (fixed?)

Time to time I see errors like this one in the Arcade CI build:

```
error NU3004: Package 'vswhere 2.6.7' from source 'https://api.nuget.org/v3/index.json': This repository indicated that all its packages are repository signed; however, this package is unsigned.
```

Doesn't make much sense to me as it happens only sometimes.. The package is definitely signed. 

Failure example:
https://dev.azure.com/dnceng/public/_build/results?buildId=172551

--- a/global.json
+++ b/global.json
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "2.1.503"
+    "dotnet": "2.2.203"
   },
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19230.6",

